modules/social_features/social_group/modules/social_group_secret/config/install/group.content_type.secret_group-group_node-topic.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.secret_group
+    - node.type.topic
+  module:
+    - gnode
+    - node
+id: secret_group-group_node-topic
+label: topic
+description: 'Adds <em class="placeholder">Topic</em> content to groups both publicly and privately.'
```

Comment:
This description probably is copy-pasted and needs to be edited. It currently refers to "publicly and privately", however content in this group will not be publicly available, correct?

Reply:
@NatashaSchon what text needs to use here?